packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -49,21 +48,28 @@ class SliderTheme extends InheritedWidget {
   /// Defaults to the ambient [ThemeData.sliderTheme] if there is no
   /// [SliderTheme] in the given build context.
   ///
-  /// Typical usage is as follows:
+  /// ## Sample code
   ///
   /// ```dart
-  /// double _rocketThrust;
-  ///
-  /// @override
-  /// Widget build(BuildContext context) {
-  ///   return new SliderTheme(
-  ///     data: SliderTheme.of(context).copyWith(activeRail: Colors.orange),
-  ///     child: new Slider(
-  ///       onChanged: (double value) => setState(() => _rocketThrust = value),
-  ///       value: _rocketThrust;
-  ///     ),
-  ///   );
+  /// class Catatonic extends StatefulWidget {
+  ///   @override
+  ///   State createState() => new CatatonicState();
   /// }
+  /// 
+  /// class CatatonicState extends State<Catatonic> {
+  ///   double _howLong;
+  /// 
+  ///   @override
+  ///   Widget build(BuildContext context) {
+  ///     return new SliderTheme(
+  ///       data: SliderTheme.of(context).copyWith(activeRailColor: const Color(0xff804040)),
+  ///       child: new Slider(
+  ///         onChanged: (double value) => setState(() => _howLong = value),
```

Comment:
trivial nit: here and below, prefer {...} rather than => for void-returning closures